spec/controllers/twilios_controller_spec.rb

```diff
@@ -8,7 +8,7 @@
 
     context 'when the call status was failed' do
       context 'but no CAB number is provided' do
-        let(:params) { { 'DialCallStatus' => 'failed' } }
+        let(:params) { Hash[params: { 'DialCallStatus' => 'failed' }] }
```

Comment:
To my eyes it's just somewhat neater / easier to parse than `let(...) {{{ ... }}}`